### PR TITLE
[WIP] - Grouping gems

### PR DESCRIPTION
**Pull request by tomas-stefano**
We discussed in a meeting about versioning, and this is the start point to group the gem in the right source to avoid conflict. The benefit is that removes the github option for some of the gems (unfortunately not all gems),

All rspec tests pass and cucumber ones too.

---

This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1438)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+  gem 'cutback_calculator', '~> 0.10'
+  gem 'debt_advice_locator', '~> 2.0'
+  gem 'debt_and_mental_health', '~> 1.0.1'
+  gem 'debt_free_day_calculator', '~> 2.1.0'
+  gem 'debt_test'
+  gem 'decision_trees', '~> 2.0.0'
+  gem 'feedback', '~> 0.3.1'
+  gem 'mortgage_calculator', '~> 1.3.5'
+  gem 'mas-assets'
+  gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
```
Comment (tomas-stefano) [inline Gemfile]: The meta-tags is not being upload to mas server. It is using a fork, so we should see what should we be done with those (but not on these PR).

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+  gem 'payday_loans_intervention', '~> 1.4'
+  gem 'pensions_calculator', '~> 1.0.0'
+  gem 'quiz'
+  gem 'rio', '= 0.0.3', source: 'http://gems.dev.mas.local'
```
Comment (tomas-stefano) [inline Gemfile]: Bundler complain about different sources, so I forced and the warnings were removed.

**Comment by misaka** [comment]
I like this change, but as discussed, we need to investigate the changes introduced by the gem updates here.

File: Gemfile [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+source 'http://gems.dev.mas.local' do
+  gem 'action_plans', '~> 4.0.1'
+  gem 'advice_plans', '~> 3.1.0'
+  gem 'agreements', '~> 2.0.1'
+  gem 'baby_cost_calculator'
+  gem 'budget_planner', '~> 4.0.3'
+  gem 'car_cost_tool', '~> 1.0.4'
+  gem 'contribution_calculator', '~> 0.0.1'
+  gem 'cost_calculator_builder', '~> 0.1.10'
+  gem 'cutback_calculator', '~> 0.10'
```
Comment (misaka) [inline Gemfile]: Should we be locking this to `~> 0.10.0`?

File: Gemfile [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+  gem 'debt_advice_locator', '~> 2.0'
```
Comment (misaka) [inline Gemfile]: Should we be locking this to `~> 2.0.0?`

File: Gemfile [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+  gem 'debt_and_mental_health', '~> 1.0.1'
+  gem 'debt_free_day_calculator', '~> 2.1.0'
+  gem 'debt_test'
+  gem 'decision_trees', '~> 2.0.0'
+  gem 'feedback', '~> 0.3.1'
+  gem 'mortgage_calculator', '~> 1.3.5'
+  gem 'mas-assets'
+  gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
+  gem 'payday_loans_intervention', '~> 1.4'
```
Comment (misaka) [inline Gemfile]: Should we be locking this to `~> 1.4.0?`

File: Gemfile (hunk 3 above)
Comment (tomas-stefano) [inline Gemfile]: Yes, we should.

**Comment by tomas-stefano** [comment]
@misaka I added the user story here: https://moneyadviceservice.tpondemand.com/entity/7261

**Pull request closed without merging**